app/models/manageiq/providers/openstack/infra_manager/host/operations.rb

```diff
@@ -0,0 +1,24 @@
+module ManageIQ::Providers::Openstack::InfraManager::Host::Operations
+  def self.included(base)
+    base.send :include, InstanceMethods
+    base.extend ClassMethods
+  end
+
+  module InstanceMethods
+    def ironic_fog_node
```

Comment:
So, it looks like the `provider_object` support is not implemented for `Openstack::InfraManager::Host`.  This could be refactored away later.

Take a look at the [provider_object_mixin](https://github.com/ManageIQ/manageiq/blob/bc548376ead317bfd7ce8d9cbcb2b97bd88ce75c/app/models/mixins/provider_object_mixin.rb).